app/mailers/rent_mailer.rb

```diff
@@ -1,11 +1,17 @@
+# frozen_string_literal: true
+
 class RentMailer < ApplicationMailer
   # Subject can be set in your I18n file at config/locales/en.yml
   # with the following lookup:
   #
   #   en.rent_mailer.rent_created.subject
   #
+
   def rent_created(rent_id)
-    @rent = Rent.find(rent_id)
-    mail to: @rent.user.email
+    I18n.with_locale(:es) do
```

Comment:
Remember to grab here the lang from the user, this should be I18n.with_locale(@rent.user.locale) do